src/scripts/main.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const tbody = document.querySelector('tbody');
+let rows = [...tbody.querySelectorAll('tr')];
+
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+
+const updateRows = () => {
+  return [...tbody.querySelectorAll('tr')];
+};
+
+const setMaxValue = (list, appendBtn, removeBtn) => {
+  if (list.length < 10) {
+    appendBtn.disabled = false;
+  } else {
+    appendBtn.disabled = true;
+  }
```

Comment:
as a suggestion, you can simplify it to:
```suggestion
  appendBtn.disabled = list.length >= 10;
```